packages/components/psammead-radio-schedule/src/utilities/index.js

```diff
@@ -0,0 +1,19 @@
+import { formatDuration } from '@bbc/psammead-timestamp-container/utilities';
+
+const getDurationFormat = (duration = '', separator = ':') => {
```

Comment:
Thanks for all this! I just think we can tweak all of these default parameters a little - `getDurationFormat` is only used in the `durationDictionary` below, correct? I think we can remove these defaults from `getDurationFormat`, as they will never be used, as `durationDictionary` has its own, that will override these every time the function is called (so the `:` separator default will never be used).